flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/assigners/ChunkSplitter.java

```diff
@@ -122,11 +122,16 @@ public ChunkSplitter(MySqlConnection jdbc, MySqlSchema mySqlSchema, int chunkSiz
         final Object[] minMaxOfSplitColumn = queryMinMax(jdbc, tableId, splitColumnName);
         final Object min = minMaxOfSplitColumn[0];
         final Object max = minMaxOfSplitColumn[1];
+        LOG.info(
+                "Start to split table {} into chunks using column {}, the min and max value are {}, {}.",
+                tableId,
+                splitColumnName,
+                min,
+                max);
```

Comment:
Maybe the number of chunks is a more important information.

Reply:
The number of chunks has been logged in method `ChunkSplitter.generateSplits`. Is it stiil need to log such detail information about table splitting?